src/@types/casperlabsSigner.d.ts

```diff
@@ -15,12 +15,13 @@ interface CasperLabsHelper {
   requestConnection: () => void;
   
   /**
-   * send base16 encoded message to plugin to sign
+   * Send Deploy in JSON format message to Signer plugin to sign.
    *
-   * @param messageBase16 the base16 encoded message that plugin received to sign
-   * @param publicKeyBase64 the base64 encoded public key used to sign the deploy, if set, we will check whether it is the same as the active key for signing the message, otherwise, we won't check.
+   * @param deploy deploy in JSON format
+   * @param sourcePublicKey base64 encoded public key used to sign the deploy
```

Comment:
The keys should be in hex format with algorithm prefix. The inline-doc suggests `base64`.